algorithms/polish.sbl

```diff
@@ -0,0 +1,240 @@
+        'by{s'}'     // 2nd person singular, e.g. czytał(a)byś
+        'by{s'}my'   // 1st person plural, e.g. czytalibyśmy
+        'by{s'}cie'  // 2nd person plural, e.g. czytalibyście
+        'by'         // 3rd person singular/plural, e.g. czytał(a)by, czytaliby
+          (delete)
+      )
+    )
+    setlimit tomark pV for ([substring]) among (
+      'asz'  'esz'   // present 2nd person singular, e.g. czytasz, piszesz
+      'amy'  'emy'   // present 1st person plural, e.g. czytamy, piszemy
+      'acie' 'ecie'  // present 2nd person plural, e.g. czytacie, piszecie
+      'cie'          // imperative 2nd person plural, e.g. czytajcie, chodźcie
+      'e{s'}{c'}'    // infinitive, e.g. przynieść
+      'a{s'}{c'}'    // infinitive, e.g. popaść
+      'a{c'}'        // infinitive, e.g. czytać
+      'e{c'}'        // infinitive, e.g. chcieć
+      'i{c'}'        // infinitive, e.g. wozić
+      '{ak}{c'}'     // infinitive, e.g. marznąć
+      'a{l/}em'       'i{l/}em' 'em'  // past 1st person singular masculine, e.g. czytałem, chodziłem, marzłem
+      'a{l/}am'       'i{l/}am' 'am'  // past 1st person singular feminine, e.g. czytałam, chodziłam, marzłam
+      'a{l/}e{s'}'    'i{l/}e{s'}'    // past 2nd person singular masculine, e.g. czytałeś, chodziłeś
+      'a{l/}a{s'}'    'i{l/}a{s'}'    // past 2nd person singular feminine, e.g. czytałaś, chodziłaś
+      'a{l/}'         'i{l/}'         // past 3rd person singular masculine, e.g. czytał, chodził
+      'a{l/}a'        'i{l/}a'        // past 3rd person singular feminine, e.g. czytała, chodziła
+      'a{l/}o'        'i{l/}o'        // past 3rd person singular neuter, e.g. czytało, chodziło
+      'ali{s'}my'     'ili{s'}my'     // past 1st person plural virile, e.g. czytaliśmy, chodziliśmy
+      'a{l/}y{s'}my'  'i{l/}y{s'}my'  // past 1st person plural nonvirile, e.g. czytałyśmy, chodziłyśmy
+      'ali{s'}cie'    'ili{s'}cie'    // past 2nd person plural virile, e.g. czytaliście, chodziliście
+      'a{l/}y{s'}cie' 'i{l/}y{s'}cie' // past 2nd person plural nonvirile, e.g. czytałyście, chodziłyście
+      'ali'           'ili'           // past 3rd person plural virile, e.g. czytali, chodzili
+      'a{l/}y'        'i{l/}y'        // past 3rd person plural nonvirile, e.g. czytały, chodziły
+      '{ak}c'      // contemporary adverbial participle (transgressive), e.g. czytając
+      // different forms of active adjectival participle, e.g. czytający (one who is reading)
+      '{ak}cy'     // nominative singular (masculine)
+      '{ak}cego'   // genitive singular (masculine/neuter)
+      '{ak}cemu'   // dative singular (masculine)
+      '{ak}cym'    // instrumental/locative singular (masculine)
+      '{ak}ca'     // nominative singular (feminine)
+      '{ak}cej'    // genitive/dative/locative singular (feminine)
+      '{ak}c{ak}'  // accusative singular (feminine)
+      '{ak}ce'     // nominative/accusative singular (neuter)
```

Comment:
_czytające_ will actually get _-ce_ removed by `remove_nouns` then `-ą` removed by `remove_general_ends`, rather than by this rule.  So this may be redundant, but perhaps _-ące_ can be exposed by removal of an ending in a previous step.